analytics_dashboard/static/apps/learners/js/spec/learners-router-spec.js

```
@@ -0,0 +1,80 @@
+define(['backbone', 'learners/js/learners-router'], function (Backbone, LearnersRouter) {
+    'use strict';
+
+    describe('LearnersRouter', function () {
+        beforeEach(function () {
+            try {
+                Backbone.history.start({silent: true});
+            } catch (e) {
+                // If Backbone.history.start() has already been
```

Call ``Backbone.history.stop`` in ``afterEach``, and then you shouldn't need to swallow these exceptions.